modules/runner/src/integrationTest/java/org/apache/ignite/internal/sql/engine/ClusterPerClassIntegrationTest.java

```diff
@@ -526,7 +526,7 @@ protected static Map<Integer, List<Ignite>> waitForIndexBuild(String tableName,
 
                 IndexStorage index = internalTable.storage().getOrCreateIndex(
                         partitionId,
-                        createIndexDescriptor(catalogTableDescriptor, catalogIndexDescriptor)
+                        create(catalogTableDescriptor, catalogIndexDescriptor)
```

Comment:
The idea was that you would use a fully qualified name instead: `IndexDescriptor.create`

Reply:
Fix it